### ISSUE TITLE
In a post, show the profile of the author at the top (either the side bar or top page) 

### ISSUE THREAD

**Issue by morchickit** [openknowledge-archive/wordpress-theme-with-bp#20]
Currently it is at the bottom, and there is missing information there. 
See example here: http://okfn.keitaro.info/2016/04/06/code-for-ghana-open-data-day-2016/

See feature from the current blog:
![screen shot 2016-05-27 at 09 56 39](https://cloud.githubusercontent.com/assets/3273659/15603207/59ea4b82-23f1-11e6-9d56-1e83b3512eb4.png)

**Comment by gocemitevski** [comment]
@pwalsh the author information are now shown at the top of the sidebar.

I've shortened the title to "About --author display name--" from "About the author: --author display name--". I hope that's OK.

I've also decreased the size of the thumbnail from 100px to 80px to improve text wrapping. Also, the avatars are now shown circled.

http://okfn.keitaro.info/2013/07/11/using-public-data-to-flag-tax-avoidance-schemes/
